motor_control/throttle/base_throttle.h

```diff
@@ -1,21 +1,36 @@
 #ifndef BASE_THR_H
 #define BASE_THR_H
 
+//#include <Servo.h>
+#define throttleIn 5   // Placeholders
+#define ESCOutput 3
+#define button 7
```

Comment:
These definitely do not belong in the base class, nor in the class that you defined above. These are parameters that the user passes your class at compile time. 

Note also that the input and output to your class is taken care of by this base class. Therefore, the only function you need to implement in your inheriting class is `compute_motor_value()`. 

Like we said on the call, there should be no changes made here to the `base_throttle.h` or `base_throttle.ino` files.